src/web/index.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2023 Kapeta Inc.
+ * SPDX-License-Identifier: MIT
+ */
+
+import { IBlockTypeProvider } from "@kapeta/ui-web-types";
+import {DesktopBlockEditorComponent} from './DesktopBlockEditorComponent';
+import DesktopBlockValidation from './DesktopBlockValidation';
+
+const blockDefinition = require('../../kapeta.yml');
```

Comment:
I had to do this 👇 but not 100% sure why. I had an issue with using json-loader for yaml files, so I changed it to `['yaml-loader']` instead of `['json-loader', 'yaml-loader']` in the webpack config that solved it https://github.com/kapetacom/provider-block-cli/blob/master/webpack.config.js#L48

```suggestion
const blockDefinition = require('../../kapeta.yml').default;
```

Reply:
I've seen that too but didn't seem to be required here (it loads just fine)